packages/syft/src/syft/service/policy/policy.py

```diff
@@ -854,6 +854,26 @@ def load_policy_code(user_policy: UserPolicy) -> Any:
 def init_policy(user_policy: UserPolicy, init_args: dict[str, Any]) -> Any:
     policy_class = load_policy_code(user_policy)
     policy_object = policy_class()
+
+    # Unwrapp {NodeIdentity : {x: y}} -> {x: y}
+    # Tech debt : For input policies, we required to have NodeIdentity args beforehand,
+    # therefore at this stage we had to return back to the normal args.
+    # Maybe there's better way to do it.
+    if len(init_args) and isinstance(list(init_args.keys())[0], NodeIdentity):
```

Comment:
Depending on how urgent this is, could we maybe fix this tech debt? I was thinking over the weekend that maybe this will affect other parts of the codebase, and the solution here is pretty dirty